### ISSUE TITLE
Small bug in the "Product scheduled for:" text

### ISSUE THREAD

**Issue by delitestudio** [woocommerce/woocommerce#27839]
The `class-wc-admin-post-types.php` has a small bug which, when a product is scheduled, opens a `<strong>` which is then closed in the wrong place.

To be precise, in the `post_updated_messages()` method this code is incorrect:

```
9  => sprintf(
	/* translators: 1: date 2: product url */
	__( 'Product scheduled for: %1$s. <a target="_blank" href="%2$s">Preview product</a>', 'woocommerce' ),
	'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), strtotime( $post->post_date ) ),
	esc_url( get_permalink( $post->ID ) ) . '</strong>'
	),
```

because it generates this text:

`<p>Product scheduled for: <strong>Ott 2, 2020 @ 10:50. <a target="_blank" href="https://xxx.com/?post_type=product&#038;p=7020</strong>">Preview product</a></p>`

As you can see the `</strong>`is closed in the `href`attribute instead of immediately after the date.

The correct code should be:

```
9  => sprintf(
	/* translators: 1: date 2: product url */
	__( 'Product scheduled for: %1$s. <a target="_blank" href="%2$s">Preview product</a>', 'woocommerce' ),
	'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), strtotime( $post->post_date ) ) . '</strong>',
	esc_url( get_permalink( $post->ID ) )
	),
```

Anyway thanks for your awesome plugin;)

**Comment by tammullen** [comment]
Hi @delitestudio 

Thank you for taking the time to report this bug, we really appreciate your help. I can reproduce it on my end. 

![Screenshot_2020-10-02_at_22_32_32](https://user-images.githubusercontent.com/24649833/94971863-ed56ea80-04ff-11eb-905a-0e315f7b4c7c.png)
![Screenshot_2020-10-02_at_22_33_02](https://user-images.githubusercontent.com/24649833/94971882-f1830800-04ff-11eb-9050-78680fa40238.png)

**Comment by sudoshreyansh** [comment]
I have fixed the issue and created a pull request #27842

**Comment by delitestudio** [comment]
Great, thanks!
